src/App.jsx

```diff
@@ -1,67 +1,71 @@
-// App.jsx
-import React, { useState } from 'react';
-import TodoItem from './TodoItem';
-import LoginComponent from './Login'; // Importa el nuevo componente de inicio de sesión
-import './App.css';
+import { useState } from "react";
+import TodoItem from "./TodoItem";
+import LoginComponent from "./Login";
+import "./App.css";
 
 const App = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const onLogin = () => {
+    setIsLoggedIn(true);
+  };
+
+  return (
+    <div className="container">
+      {isLoggedIn ? <TodoList /> : <LoginComponent onLogin={onLogin} />}
+    </div>
+  );
+};
+
+function TodoList() {
```

Comment:
one option is to split the `<App />` component and create an aux component named `<TodoList />` that handles everything related to the todo list while `<App />` handles just the "is the user logged in?" thing,